web/src/unread_ops.ts

```diff
@@ -544,7 +545,7 @@ export function process_unread_messages_event({
         !message_lists.current.can_mark_messages_read() &&
         message_lists.current.has_unread_messages()
     ) {
-        unread_ui.notify_messages_remain_unread();
+        message_list_navigation.update();
```

Comment:
The refactor to have a `message_list_navigation.update` function has appeared in both versions of this; if we keep doing iterations on this, maybe we make a prep commit we can merge to reduce overall churn.